software/edge/src/aeris_orchestrator/aeris_orchestrator/mavlink_adapter.py

```
@@ -0,0 +1,225 @@
+                        self._target_system,
+                        self._target_component,
+                        index,
+                        mavutil.mavlink.MAV_FRAME_LOCAL_NED,
+                        mavutil.mavlink.MAV_CMD_NAV_WAYPOINT,
+                        1 if index == 0 else 0,
+                        1,
+                        0.0,
+                        0.0,
+                        0.0,
+                        0.0,
+                        int(round(north_m * _METERS_TO_CENTIMETERS)),
```

Ruff (0.14.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)